src/Kernel/Magic/Resolution/MagicResolver.cs

```diff
@@ -125,14 +136,7 @@ public IEnumerable<MagicSymbol> FindMagic(AssemblyInfo assm)
                         .Where(t =>
                         {
                             if (!t.IsClass && t.IsAbstract) { return false; }
-                            var matched = t.IsSubclassOf(typeof(MagicSymbol));
-
-                            // This logging statement is expensive, so we only run it when we need to for debugging.
-                            #if DEBUG
```

Comment:
I was running in Debug mode, so this could explain why it was eating so much of my resources.
However, this is very useful debug information, so maybe we should keep it?

Reply:
Makes sense, added it back in!